app-utils/wsl-drop-caches/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=wsl-drop-caches
+PKGSEC=misc
+PKGDES="A tool reclaiming cached memory in WSL2 periodically if idle"
```

Comment:
A tool for reclaiming idle cached memory periodically in WSL2.

Reply:
descr. editted